doc/source/libraries/xorbits_data/lightgbm_example.rst

```diff
@@ -0,0 +1,103 @@
+.. _10min_lightgbm:
```

Comment:
I think this `lightgbm_example.rst` file is redundant. Just keep the `lightgbm.rst` under `xorbits_train` dir.
And update the `index.rst` in  `xorbits_train` dir to link the `lightgbm.rst`.